README.md

```diff
@@ -76,27 +71,73 @@ For a more detailed overview and getting started guide, please see
    these databases. See the [SQL](#sql) section for more details.
 * **GraphQL Language Features:**  We prioritized and implemented a subset of all functionality supported by the GraphQL language. We hope to add more functionality over time.
 
-## Querying OrientDB with GraphQL
+## Getting Started
```

Comment:
This may be personal preference, but if there are x steps I need to take towards getting started, I would prefer they would either in an ordered list and/or in distinct sections - kind of like treating it as a checklist. The former (enumerated or bulleted list) is probably the better option here since distinct sections should probably only get used when you have more complicated and/or detailed steps.

Reply:
I am not sure how to add the ordered list without being too verbose. If added the ordered list, I would still have to have comments in the code that separated it into different steps.

Reply:
I think my main reservation with this section is that it doesn't actually help with getting started - there's no install, config, etc process. This section tells me that I need a GraphQL schema and that I will need a query. This is more of a this-is-what-you'll-need section.

Reply:
I agree with your comment. I am going to refactor this section.

Reply:
I removed the "Getting Started" section for now and reduced the scope of this PR.